xds/src/main/java/io/grpc/xds/internal/security/certprovider/CertProviderSslContextProvider.java

```diff
@@ -83,6 +84,8 @@ protected CertProviderSslContextProvider(
     } else {
       rootCertHandle = null;
     }
+    this.isUsingSystemRootCerts = CommonTlsContextUtil.isUsingSystemRootCerts(
```

Comment:
This needs to factor in `rootCertInstance == null` as well, right? (I see that the two usages don't care about that case, but it's asking for a future bug.)

Reply:
Done.